### ISSUE TITLE
Doubts about the parameter in Vref Target-mu

### ISSUE THREAD

**Issue by 343333333** [shankar1729/jdftx#197]
Dear developers,
    i was a little confused about Vref in target-mu. As you mentioned in #113 ,The Vref can be defined as `the solvation model papers based on PZC of metal surfaces` . So , if i want to get good agreement with the experiment ,is it a possibly way to choose the PZC(or the working function) as the Vref? which means that , if the surface working function in experiment is 3.94V and the pzc Vs RHE is -0.07V, could i set the Vref as (3.94+0.07)=3.81V Vs RHE(directly in  experiment condition)? because when i used the CANDLE model (whose vref is 4.66V) , the Voltage phenomenon get good agreement with the experiment, but all the onset potential have shifted a constant V(in every elementary reaction ). So is that a good way to eliminate the system error?
   looking forward to your reply,
   thanks

**Comment by shankar1729** [comment]
Vref is a calibration of the electron chemical potential scale against the experimental standard electrodes, which should not depend on the specific electrode you are working with. In contrast, PZC is a property of each electrode. Therefore these are two independent quantities and cannot reasonably be equated (unless the PZC of an electrode happens to be close to 0V SHE by accident).

Now, we could make different choices for how Vref is calibrated, which could lead to different systematic errors. We have typically used the solvation model predicted PZC vs experimental PZC calibration to cancel any systematic errors in the dipole layer at metal-liquid interfaces. An alternate choice is based on the computational hydrogen electrode i.e. considering the energetics of H+ to H2 within DFT, which leads to a number ~ 4.3 eV.

Ultimately, DFT often relies on cancellation of errors for reasonable results. If you can determine that one of these schemes consistently gives you the best error cancellation, then use it with reasonable justification. For example, in HER catalysis calculations, the CHE reference may give you the best cancellation since it also involves the same reaction. However this cancellation relies on DFT errors in hydrogen and similar bonding in small organics. We found better agreement for Cu UPD on Pt (in the 2017 JCP paper on GC-DFT) using the PZC-based solvation model reference, probably because that involved primarily metallic surfaces and not energetics of covalent bonds.

In summary, there is no obvious best scheme for Vref, and there are two common choices outlined above. However, setting Vref = PZC of the specific surface in question does not make sense.

Hope that helps!
Best,
Shankar

**Comment by 343333333** [comment]
Hi Shankar,
Thank you for being so patient in answering my questions. Now I know how to eliminate systematic errors. But I still have some questions about the actual operation, I would appreciate it if you could give me some guidance about that.
As you mentioned ,we could use the PZC between the experiment and the theory to eliminate that . I did find in my calculations that when I set the target-mu=-0.1713(which means the V =0 Vs SHE in CANDLE model), the net electron does not equal 0（nElectrons=206.2773 but the total number of  pseudopotential electrons is 207).  So, if i do a regular ele-scf without the parameter target-mu and get the target-mu equals X, could I say this is the PZC Vs SHE in theory? 
And I use the equation in your tutorial in  [https://jdftx.org/FixedPotential.html](url)
 mu=X-Vset/27.2114 to regulate applied voltage. To connect the experiment and the theory, if the PZC is -0.07V in experiment, then the Vtrue equals Vset+0.07 .Is that so? 
Last, for the ‘`An alternate choice is based on the computational hydrogen electrode i.e. considering the energetics of H+ to H2 within DFT, which leads to a number ~ 4.3 eV`.’ , Could you tell where can I find the theory of this value 4.28 , I want to know more about the theory.
Looking forward to your reply and thanks again for you patient answering!
Best,
Benjamin

**Comment by shankar1729** [comment]
Hi Benjamin,

Yes, the calculation without target-mu (and with no initial-charge) is a neutral calculation, and therefore corresponds to the PZC. The mu that you get this way is the PZC of that specific surface.

In the remainder of what you wrote, you have to be clear about what the potential is relative to. I'm not sure what you mean by Vtrue here. It should be a lot simpler than this. A given experimental potential relative to SHE corresponds to a specific mu for the electrons; you use that mu in target-mu. The only questions is what Vref you use for SHE in this conversion.

If you want to measure V - PZC, then compute the PZC using the neutral calculation as above, and then set various mu at required offsets from that neutral mu.

See [this paper](https://pubs.acs.org/doi/10.1021/jp100402x) for details on the 4.28V reference.

Best,
Shankar

**Comment by 343333333** [comment]
Dear Shankar,

Thanks for sharing, and sorry for the ambiguous expression of Vtrue.

What I want to express is whether I can correct a V Vs PZC (in JDFTX) through PZC (in Experiment), because both of them mean that the net charge on the surface is 0 in the physical. For example, when I set -1V Vs PZC in JDFTx, if the experimental PZC is -0.07V, can I say that the potential in -1V Vs pzc(in jdftx) is the same as the potential  -1.07V Vs PZC (in experiment)? Because I think they're both shifting from the same physical meaning to the same physical quantity. Dose it make any sense? 

Best,
Benjamin

**Comment by shankar1729** [comment]
I see. No: I would expect V - PZC to be the most reliable quantity to transfer between computation and experiment. So I would expect -1V theoretical V - PZC to best correspond  to -1V experimental V - PZC, even if the theoretical and experimental PZCs disagree somewhat on an absolute scale. 

Best,
Shankar

**Comment by 343333333** [comment]
OK, i will kept that in mind.
Thanks for relying.

Best,
Benjamin
